cli/tools/upgrade.rs

```diff
@@ -210,30 +210,96 @@ pub fn check_for_upgrades(
 
   // Print a message if an update is available
   if let Some(upgrade_version) = update_checker.should_prompt() {
+    let auto_upgrade_env = match env::var("DENO_AUTO_UPGRADE") {
+      Ok(env_var) => env_var,
+      Err(_) => String::from(""),
+    };
```

Comment:
I think it would be enough to do `env::var("DENO_AUTO_UPGRADE).is_ok()`, no need to probe for a specific value

Reply:
I actually was thinking if we could make this feature more fine grained, like different levels for `MAJOR`, `MINOR`, or `PATCH` so followed this path, let me know if that seems unnecessary and I'll change it to the above syntax.

Reply:
Since testing it is hard I would suggest to go with the simpler solution for now - if this feature proves popular we can always add additional functionality.